packages/cmd/src/CommandProcessor.ts

```diff
@@ -923,10 +926,14 @@ export class CommandProcessor {
             args: params.arguments,
             silent: (params.silent == null) ? false : params.silent,
             responseFormat: params.responseFormat,
-            stream: params.arguments.stream
+            stream: ImperativeConfig.instance.daemonContext?.stream
         });
     }
 
+    private getStdinStream(): stream.Readable {
+        return this.mDaemonContext?.stdinStream || process.stdin;
+    }
+
```

Comment:
I do appreciate having this getter for the stream.
I know it's just private, but could we add some JSDoc?